test/.gitignore

```diff
@@ -1 +1,2 @@
 !*.zip
+/0/
```

Comment:
`mvn test -Dtest=OldDataMonitorTest`

output:

```
ip-192-168-1-111:test recena$ cd 0/
ip-192-168-1-111:0 recena$ ls -la
total 8
drwxr-xr-x  3 recena  staff  102 26 abr 18:31 .
drwxr-xr-x  8 recena  staff  272 26 abr 18:31 ..
-rw-r--r--  1 recena  staff  197 26 abr 18:31 build.xml
ip-192-168-1-111:0 recena$ 
```

This is being generated and hence, it should be ignored.